src/Kovan-DssSpell.t.sol

```diff
@@ -542,58 +606,84 @@ contract DssSpellTest is DSTest, DSMath {
+            if (!values.collaterals[ilk].aL_enabled) {
+                assertTrue(aL_line == 0);
+                assertEq(line, normalizedTestLine);
+                assertTrue((line >= RAD && line < BILLION * RAD) || line == 0);  // eq 0 or gt eq 1 RAD and lt 1B
+            } else {
+                assertTrue(aL_line > 0);
+                assertEq(aL_line, values.collaterals[ilk].aL_line * RAD);
+                assertEq(aL_gap, values.collaterals[ilk].aL_gap * RAD);
+                assertEq(aL_ttl, values.collaterals[ilk].aL_ttl);
+                assertTrue((aL_line >= RAD && aL_line < BILLION * RAD) || aL_line == 0);  // eq 0 or gt eq 1 RAD and lt 1B
+            }
+            uint256 normalizedTestDust = values.collaterals[ilk].dust * RAD;
+            assertEq(dust, normalizedTestDust);
+            assertTrue((dust >= RAD && dust < 10 * THOUSAND * RAD) || dust == 0); // eq 0 or gt eq 1 and lt 10k
+            }
+            {
+            (, uint256 chop, uint256 dunk) = cat.ilks(ilk);
+            // Convert BP to system expected value
+            uint256 normalizedTestChop = (values.collaterals[ilk].chop * 10**14) + WAD;
+            assertEq(chop, normalizedTestChop);
+            // make sure chop is less than 100%
+            assertTrue(chop >= WAD && chop < 2 * WAD);   // penalty gt eq 0% and lt 100%
+            // Convert whole Dai units to expected RAD
+            uint256 normalizedTestDunk = values.collaterals[ilk].dunk * RAD;
+            assertEq(dunk, normalizedTestDunk);
+            // put back in after LIQ-1.2
+            assertTrue(dunk >= RAD && dunk < MILLION * RAD);
+            }
+            {
+            (,uint256 mat) = spot.ilks(ilk);
+            // Convert BP to system expected value
+            uint256 normalizedTestMat = (values.collaterals[ilk].mat * 10**23);
+            assertEq(mat, normalizedTestMat);
+            assertTrue(mat >= RAY && mat < 10 * RAY);    // cr eq 100% and lt 1000%
+            }
+            {
+            (address flipper,,) = cat.ilks(ilk);
+            FlipAbstract flip = FlipAbstract(flipper);
+            // Convert BP to system expected value
+            uint256 normalizedTestBeg = (values.collaterals[ilk].beg + 10000)  * 10**14;
+            assertEq(uint256(flip.beg()), normalizedTestBeg);
+            assertTrue(flip.beg() >= WAD && flip.beg() < 105 * WAD / 100);  // gt eq 0% and lt 5%
+            assertEq(uint256(flip.ttl()), values.collaterals[ilk].ttl);
+            assertTrue(flip.ttl() >= 600 && flip.ttl() < 10 hours);         // gt eq 10 minutes and lt 10 hours
+            assertEq(uint256(flip.tau()), values.collaterals[ilk].tau);
+            assertTrue(flip.tau() >= 600 && flip.tau() <= 3 days);          // gt eq 10 minutes and lt eq 3 days
+
+            assertEq(flip.wards(address(cat)), values.collaterals[ilk].liquidations);  // liquidations == 1 => on
+            // assertEq(flip.wards(address(makerDeployer06)), 0); // Check deployer denied
```

Comment:
Any reason these checks are commented out? (Do we retain control in Kovan or something?)

Reply:
I commented them for now as in kovan there wasn't a declared `makerDeployer06`.
Actually we should list all the different deployers we have had so far.